### DISCUSSION TITLE
Como conseguir badge yolo

### DISCUSSION THREAD

**Discussion by VHAA1718** [community/community#20756] [Other Feature Feedback, Questions, & Ideas] [answered]
Como conseguir badge yolo

**Comment by VictorHugoAguilar** [comment, marked as answer]
Solo necesitas:
Abra su repositorio; Crear nueva rama; Haga algunas actualizaciones; A continuación, cree una solicitud de extracción a la rama principal; ¡IMPORTANTE! Añadir revisor. Para hacer esto, debe agregar acceso al repositorio de alguien, luego agregarlo como revisor; Después de esto, simplemente haga clic en combinar como lo hace siempre Revisa tu perfil y ve el logro de yolo.

#19530
